src/SSHDebugPS/packages.config

```diff
@@ -4,4 +4,6 @@
   <package id="Microsoft.VisualStudio.Imaging" version="15.0.26606" targetFramework="net45" userInstalled="true" />
   <package id="Microsoft.VisualStudio.Utilities" version="15.0.26606" targetFramework="net462" />
   <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net45" />
+  <package id="VSSDK.DTE" version="7.0.4" targetFramework="net461" />
```

Comment:
>VSSDK.DTE [](start = 15, length = 9)

I think you want this guy instead: [https://www.nuget.org/packages/EnvDTE/](https://www.nuget.org/packages/EnvDTE/). `VSSDK.DTE` is probably fine, but it isn't the official copy.

Reply:
@andrewcrawley gave me this one. I'll update it.